authentication/account/repository/identity_blackbox_test.go

```diff
@@ -36,13 +38,59 @@ func (s *IdentityRepositoryTestSuite) TestDelete() {
 		assert.Nil(t, err)
 		identities, err := s.Application.Identities().List(s.Ctx)
 		require.NoError(t, err, "Could not list identities")
-		require.True(t, len(identities) >= 1)
+		require.NotEmpty(t, identities)
 		// make sure that the deleted identity is not part of the result
 		for _, identity := range identities {
 			assert.NotEqual(t, identity1.ID(), identity.ID)
 		}
 	})
 
+	s.T().Run("ok hard delete by identity ID", func(t *testing.T) {
+		// given
+		g := s.NewTestGraph(t)
+		identity1 := g.CreateIdentity()
+		// create a second identity
+		g.CreateIdentity()
+
+		unscoped := func(s *gorm.DB) *gorm.DB {
+			return s.Unscoped()
```

Comment:
This variable name `s *gorm.DB` mixed up with `s` for suite .. renaming this variable would make things clearer.

Reply:
addressed in df44f17